tests/e2e/web/specs/signIn.preSetup.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test as base, expect } from '@playwright/test';
+import { SignInPage } from '../pages/signInPage';
+import { config } from '../TESTING_CONFIG';
+
+//Baisc login using fixture created from signInPage
+
+const test = base.extend<{ signInPage: SignInPage }>({
+  signInPage: async ({ page }, use) => {
+    const signInPage = new SignInPage(page);
+    await use(signInPage);
+  },
+});
+
+test('Signin test', async ({ signInPage, page }) => {
+  await test.step('Navigated to login page', async () => {
+    await page.goto('/');
+  });
+  await test.step(
+    'Logging in '.concat(page.url(), ' as ', config.DEFAULT_LOGIN),
+    async () => {
+      await page.waitForLoadState('networkidle');
+      await signInPage.clickSignInText();
+      await signInPage.fillEmailField(config.DEFAULT_LOGIN);
+      await signInPage.fillPasswprdField(config.DEFAULT_PASSWORD);
+      await page.waitForLoadState('networkidle');
+      await signInPage.clickSignInWithEmailButton();
+      await expect(page.getByText(/Sign in to your account/)).not.toBeVisible();
```

Comment:
This fails for some reason. When it clicks on the button, it's supposed to create the user and authenticate by storing the creds in the indexedDB. That may be the cause of the issue (I haven't debugged).